packages/web-components/src/components/va-official-gov-banner/va-official-gov-banner.scss

```diff
@@ -0,0 +1,304 @@
+:host {
+  display: block;
+}
+
+.banner {
+  background-color: #f0f0f0;
```

Comment:
All of the hex colors hardcoded are from the USWDS v3 example. When this component is updated to use the v3 modules (ie. `@use "usa-banner"`), these colors should automatically map to the relevant classes in the markup and can eventually removed. I did not think it was necessary to add these to the `css-library` package for that reason.